src/job-execution/Sas9JobExecutor.ts

```diff
@@ -18,6 +19,7 @@ export class Sas9JobExecutor extends BaseJobExecutor {
     serverUrl: string,
     serverType: ServerType,
     private jobsPath: string,
+    private requestClientSingle: RequestClient,
```

Comment:
can't it be renamed to just requestClient? What's the need for adding Single in variable name?